stdlib/datetime.pyi

```diff
@@ -265,6 +265,7 @@ class datetime(date):
         def fromtimestamp(cls, __timestamp: float, tz: _TzInfo | None = ...) -> Self: ...
 
     @classmethod
+    @deprecated("Use timezone-aware objects to represent datetimes in UTC; e.g. by calling .fromtimestamp(datetime.UTC)")
```

Comment:
This and utcnow() were always mildly discouraged and the replacement has always been available, so chose to deprecate across versions.